person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id, :parent_permission
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Kindly consider making the `of_age?` method private. This approach keeps the method's logic internal to the class and prevents unnecessary exposure to external code. Since `of_age?` is used to determine age-related conditions, making it private would improve the overall organization of your code. This way, you maintain a cleaner and more focused interface for users of the `Person` class without revealing implementation details.

Reply:
well resolved! ✔️